docs/backend/deprecation.md

```diff
@@ -0,0 +1,314 @@
+---
+myst:
+  html_meta:
+    "description": "A guide how to do deprecations, including Python, ZCML and templates in Plone."
+    "property=og:description": "A guide how to do deprecations, including Python, ZCML and templates in Plone."
+    "property=og:title": "Deprecation"
+    "keywords": "deprecation, zcml, template, jbot"
+---
+
+(backend-deprecation-label)=
+# Deprecation
+
+## Introduction
+
+This document describes rationales, configuration and best practices of deprecations in Plone, Zope and Python.
+It is meant as a styleguide on how to apply deprecations in Plone core packages.
+It also has a value as a general overview on how to deprecate in Python.
+
+
+### Why Deprecation
+
+At some point we
+
+- need to get rid of old code,
+- want to unify api style (consistent api),
```

Comment:
```suggestion
- want to unify API style (consistent API),
```

Reply:
I saw below in other parts of the document that `API`, was uppercased.
For consistency I would always use it uppercased.
I also think it is more correct to have it uppercased.